test/functions/iterators/ldelaney/dynamicIterStrideable.chpl

```diff
@@ -0,0 +1,18 @@
+use DynamicIters;
+proc main(){
+	var r1 = 1..10 by 2;
+	var r2 = 7..8;
+	var r3 = -2..-1;
+	var d1 = {r1, r2, r3};
+	var bigArr : [r1][r2][r3] int;
```

Comment:
You don't need to change this, but for future reference you can write this as:

```chpl
var bigArr : [d1] int;
```

To get a multidimensional array.